minho/programmers/Level_1/7일차/신규_아이디_추천.js

```diff
@@ -0,0 +1,22 @@
+function solution(new_id) {
+    new_id = new_id
+        .toLowerCase() // 1단계
+        .replace(/[~!@#$%^&*()=+:?,<>/\[\]\{\}]/g, '') // 2단계
+        .replace(/\.{2,}/g, '.') // 3단계
+        .replace(/\.$|^\./g,''); // 4단계
+    console.log(new_id)
+    
+    if(!new_id){ // 5단계
+        new_id = 'a'; 
+    } 
+    if(new_id.length > 15){ // 6단계
+        new_id = (new_id
+            .substr(0, 15))
+            .replace(/\.$|^\./g,''); // 4단계 반복
+    }
```

Comment:
```suggestion
    .slice(0, 15).replace(/\.$/, '');
```
slice메서드를 이용해서 푸는 방법도 있다 라는것을 알려드리고 싶었습니다! 너무 잘푸셨어요